examples/tensorflow-privacy/README.md

```diff
@@ -27,40 +29,35 @@ This will create a new directory called `tensorflow-privacy` containing the foll
 -- README.md
 ```
 
-### Installing dependencies
-
-Project dependencies are defined in `pyproject.toml`. Install them with:
-
 ```shell
-pip install .
+tensorflow-privacy
+├── tf_privacy
+│   ├── client_app.py   # Defines your ClientApp
+│   ├── server_app.py   # Defines your ServerApp
+│   └── task.py         # Defines your model, training, and data loading
+├── pyproject.toml      # Project metadata like dependencies and configs
+└── README.md
 ```
 
-## Run Flower with tensorflow-privacy and TensorFlow
+### Note on the Code (Privacy Hyperparameters)
 
-### 1. Start the long-running Flower server (SuperLink)
-
-```bash
-flower-superlink --insecure
-```
+Please note that, at the current state, users cannot set `NodeConfig` for simulated `ClientApp`s. For this reason, the hyperparameter `noise_multiplier` is set in the `client_fn` method based on a condition check on `partition_id`. This will be modified in a future version of Flower to allow users to set `NodeConfig` for simulated `ClientApp`s.
```

Comment:
```suggestion
> [!NOTE]
>Please note that, at the current state, users cannot set `NodeConfig` for simulated `ClientApp`s. For this reason, the hyperparameter `noise_multiplier` is set in the `client_fn` method based on a condition check on `partition_id`. This will be modified in a future version of Flower to allow users to set `NodeConfig` for simulated `ClientApp`s.
```

Reply:
this will make it render more nicely (as a note as at the end of this readme: https://github.com/adap/flower/tree/main/examples/quickstart-pytorch).